CHANGELOG.md

```diff
@@ -1,11 +1,37 @@
+## 1.5.1 (YYYY-MM-DD)
+
+### Breaking Changes
+* None.
+
+### Enhancements
+* None.
+
+### Fixed
+* None.
```

Comment:
We should probably add the fixes similar to JS: https://github.com/realm/realm-js/blob/master/CHANGELOG.md#fixed-1